server/jetstream_test.go

```diff
@@ -22615,6 +22615,51 @@ func TestJetStreamConsumerPauseViaEndpoint(t *testing.T) {
 	})
 }
 
+func TestJetStreamConsumerPauseResumeViaEndpoint(t *testing.T) {
+	s := RunBasicJetStreamServer(t)
+	defer s.Shutdown()
+
+	nc, js := jsClientConnect(t, s)
+	defer nc.Close()
+
+	_, err := js.AddStream(&nats.StreamConfig{
+		Name:     "TEST",
+		Subjects: []string{"TEST"},
+	})
+	require_NoError(t, err)
+
+	_, err = js.AddConsumer("TEST", &nats.ConsumerConfig{
+		Name: "CONSUMER",
+	})
+	require_NoError(t, err)
+
+	getConsumerInfo := func() ConsumerInfo {
```

Comment:
This can just be `js.ConsumerInfo("TEST", "CONSUMER")`

Reply:
Did not use it since it seemed to be missing the new `Paused` flag.
Believe that would be because the go client is not updated to support that yet, or am I missing something?

Reply:
Yes you are correct! Apologies.

Reply:
No worries! Just checking 😄